android-easylocation/src/main/java/com/akhgupta/easylocation/EasyLocationClientFactory.java

```diff
@@ -0,0 +1,83 @@
+package com.akhgupta.easylocation;
+
+import android.Manifest;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.location.LocationManager;
+import android.support.v4.content.ContextCompat;
+
+import com.google.android.gms.common.ConnectionResult;
+import com.google.android.gms.common.GoogleApiAvailability;
+
+public class EasyLocationClientFactory {
+    private Context context;
+    private static EasyLocationClient currentEasyLocationClient;
```

Comment:
@timhuff static fields can create memory leaks. Android lint is giving warnings for these too